packages/react-dom/src/__tests__/ReactDOMFizzStaticNode-test.js

```diff
@@ -22,6 +20,8 @@ describe('ReactDOMFizzStaticNode', () => {
     React = require('react');
     if (__EXPERIMENTAL__) {
       ReactDOMFizzStatic = require('react-dom/static');
+      global.AbortController =
+        require('abortcontroller-polyfill/dist/cjs-ponyfill').AbortController;
```

Comment:
AbortController in Node environments has inconsistent behavior, for example in versions v.15.14.0 ~ v16.13.1, AbortSignal.abort() does not support the "reason" parameter, but in version v16.16.0 and above it has been tested to support the "reason" parameter. Therefore, using "abortcontroller-polyfill" ensures consistent behavior across all Node versions, helping to maintain compatibility.

Reply:
Can we somehow hook into the Jest's standard node environment and make sure we always do this? Seems like this is something we'll run into frequently otherwise.

Reply:
![image](https://user-images.githubusercontent.com/22427967/217217003-1c02e811-e5e3-4fdd-adae-f5ea74c60155.png)

Okay, I'll add the abortcontroller-polyfill in the scripts/jest/setupEnvironment.js file, so we can make sure it's always effective in Jest's standard node environment.